### DISCUSSION TITLE
limit the options to ones that are available to a specific view [Mar 05]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5284] [Discord Threads]
> 🙋‍♂️ **bassianotis**  
## Alternatively, is there a way to provide a form, but limit the options to ones that are available to a specific view? Here is a screenshot of Airtable offering this option.  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1070831471931170896/Screenshot_from_2023-02-02_14-18-51.png)

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### if you share a base to a user, then he/she should see all the views.   
#### if you share a view to a user, then he/she should only see that view. (however it s read-only atm)  
#  
> 🙋‍♂️ **replavar**  
#### Not completely correct - the Form view is not read-only but the Grid view (sadly) is  
#  
> 🙋‍♂️ **wkw**  
#### oh yes. thanks for supplementing that.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1070831472283488296)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
